src/linear_algebra/matrix/reindex.lean

```diff
@@ -31,112 +32,120 @@ open_locale matrix
 
 
 
-variables {l m n : Type*} [fintype l] [fintype m] [fintype n]
-variables {l' m' n' : Type*} [fintype l'] [fintype m'] [fintype n']
+variables {l m n o : Type*} [fintype l] [fintype m] [fintype n] [fintype o]
+variables {l' m' n' o' : Type*} [fintype l'] [fintype m'] [fintype n'] [fintype o']
 variables {m'' n'' : Type*} [fintype m''] [fintype n'']
-variables {R : Type*}
+variables (R A : Type*)
```

Comment:
I see why `R` has to be explicit, but why make `A` explicit too?
```suggestion
variables (R : Type*) {A : Type*}
```

Reply:
Because it's very hard to state `reindex_linear_equiv_symm` if `A` is implicit, and generally hard to use the equiv in any unapplied context.

Reply:
OK, I agree inserting one `_` in many places is better than inserting `@` and many `_`s in a few places.